dash/App/Accounts/Add/AddAddress/index.js

```diff
@@ -11,7 +11,6 @@ class AddAddress extends React.Component {
       index: 0,
```

Comment:
is this the right file? I ran these changes locally and didnt see any of the code getting run